### PR TITLE
Astro 12/13 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://docs.google.com/spreadsheets/d/1O46c5fZkirTzop_uHL7y0nreFbakmAomgOxPaBeU_So/edit#gid=1773722751

Base doc updates to follow in the morning

### CODE REVIEW

File: astro/airflow-api.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Updates to the Airflow REST API are released in new Airflow versions and new rel
 
 Calling the Airflow REST API for a Deployment requires:
 
-- An Astro access token.
+- A Deployment API key. See [Deployment API keys](api-keys.md).
```
Comment (jwitz) [inline astro/airflow-api.md:32]:
```suggestion
- An Astro access token.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -17,11 +17,46 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- `n2-highcpu-16`
 
 ### Additional improvements 
 
-- The maximum possible **Worker Count** for worker queues has been increased from 30 to 100.
+- In the **Clusters** tab of the Cloud UI, you can now click a cluster entry to see details about the cluster configuration, including which **Worker Types** are enabled for the cluster.
+- You can now see your Deployment's **ID** in the **Deployments** menu and on the Deployment information screen in the Cloud UI. A Deployment ID is required for deploying code using CI/CD.
+- You can now access your Organization's **OpenLineage URL** from the **Settings** page of the CLoud UI.
+- Workspaces are now sorted alphabetically in the Cloud UI.
+- [Potential release note for custom Kubernetes tags on clusters]
```
Comment (jwitz) [inline astro/release-notes.md:51]: Can someone help me decide whether this change needs a release note? https://github.com/astronomer/astro/issues/6093

File: astro/release-notes.md [hunk 3]
```diff
@@ -17,11 +17,46 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- Committing to GitHub and configuring a GitHub repository now happen in two separate menus dedicated to each function.
+- The default CI/CD pipeline included in the Cloud IDE project supports DAG-only deploys. Using the CI/CD pipeline to deploy DAG changes to Astro is now significantly faster. 
+- The GitHub configuration menu now includes **Clone Repo** settings. You can use these settings to make files in your configured GitHub repo accessible when you run DAGs in the Cloud IDE, such as helper functions in an `include` folder. 
+- You can now explicitly mark upstream dependencies for a cell from the cell's configuration menu.
+
+### Support for n2 GCP instances
+
+You can now configure worker queues with the following `n2` node types on GCP:
+
+- `n2-standard-4`
```
Comment (jwitz) [inline astro/release-notes.md:35]: @afastronomer Can you please help me get the actual available resource figures for these worker types? Thank you!

File: astro/release-notes.md [hunk 4]
```diff
@@ -17,11 +17,46 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+## December 13, 2022 
+
+### Improvements to the Cloud IDE
+
+The [Cloud IDE](cloud-ide/overview.md) includes several new features which improve DAG authoring and testing:
+
+- Committing to GitHub and configuring a GitHub repository now happen in two separate menus dedicated to each function.
+- The default CI/CD pipeline included in the Cloud IDE project supports DAG-only deploys. Using the CI/CD pipeline to deploy DAG changes to Astro is now significantly faster. 
+- The GitHub configuration menu now includes **Clone Repo** settings. You can use these settings to make files in your configured GitHub repo accessible when you run DAGs in the Cloud IDE, such as helper functions in an `include` folder. 
+- You can now explicitly mark upstream dependencies for a cell from the cell's configuration menu.
```
Comment (jlaneve) [inline astro/release-notes.md:29]: Can we also include dag-based deploys? Our default CI/CD is now set to use dag-based deploys if you're only committing DAGs.

cc @paolaperaza - think it's worth including?

File: astro/release-notes.md (hunk 3 above)
Comment (dnarain-astro) [reply astro/release-notes.md:35]: We also added support for c2 instances ([PR](https://github.com/astronomer/astro/pull/6612)) - don't know how that didn't get flagged for inclusion in the sheet, but I can see these instances in stage right now.  Specifically, we also allow c2-standard-4 and c2-standard-8 instances for worker queues.

Comment (nkapinos) [reply astro/release-notes.md:35]: @dnarain-astro that PR was from last month.  These changes are for this week's release only.

**Review by nkapinos — APPROVED** [review]
LGTM

File: astro/release-notes.md (hunk 3 above)
Comment (jwitz) [reply astro/release-notes.md:35]: @dnarain-astro Nevertheless good to know now for our reference docs. Do you have the figures for how much CPU and memory are available in those instances after Astro overhead is accounted for?

File: astro/release-notes.md (hunk 2 above)
Comment (jwitz) [inline astro/release-notes.md:51]:
```suggestion
```

Comment (jwitz) [reply astro/release-notes.md:51]: Removing for now, can add later as this seems like an optional feature and not an automatic change to behavior

**Pull request merged**